content/influxdb/cloud-iox/union.md

```diff
@@ -0,0 +1,74 @@
+---
+title: The UNION clause
+list_title: The UNION clause
+description: > 
+    Use the `UNION` clause to combine the results of two or more queries into a single set of results.
+menu:
+  influxdb_cloud_iox:
+    name: The UNION clause
+    parent: Explore data using SQL
+weight: 295
+---
+
+The `UNION` clause combines the results of two or more `SELECT` statements into a single results set.
+
+Key points to keep in mind when using the `UNION` clause:
+
+ - the number of columns in the column list must be the same
+ - the columns in the column list must be in the same order
+ - the data types in the column list must be the same or compatible  
+
+- [Syntax](#syntax)
+- [Examples](#examples)
+
+## Syntax
+
+```sql
+SELECT a, b, c
+FROM measurement_1
+UNION [ALL]
+SELECT a, b, c
+FROM measurement_2
+``` 
+
```

Comment:
Explain the difference between `UNION` and `UNION ALL`: https://dataschool.com/learn-sql/what-is-the-difference-between-union-and-union-all/#:~:text=The%20main%20difference%20between%20UNION,keeps%20all%20records%2C%20including%20duplicates